frontend/src/pages/IstioConfigNew/IstioConfigNewPage.tsx

```diff
@@ -105,6 +112,14 @@ type State = {
 
 const formPadding = kialiStyle({ padding: '30px 20px 30px 20px' });
 
+const editIcon = kialiStyle({ marginLeft: '0.35em', marginBottom: '0.2em' });
```

Comment:
In my opinion I think rem measures are more robust because it depends on global font size instead of parent one, but if you like em is valid too ;-)